pkg/service/backup/restore_worker_data.go

```diff
@@ -57,6 +58,28 @@ func (w *restoreWorker) restoreData(ctx context.Context, run *RestoreRun, target
 	return nil
 }
 
+func (w *restoreWorker) initRemainingBytesMetric(ctx context.Context, run *RestoreRun, target RestoreTarget) {
+	for _, location := range target.Location {
+		var restorePerLocationDataSize int64
+		err := w.forEachRestoredManifest(
+			ctx,
+			location,
+			func(miwc ManifestInfoWithContent) error {
+				return miwc.ForEachIndexIterWithError(
+					target.Keyspace,
+					func(fm FilesMeta) error {
+						restorePerLocationDataSize += fm.Size
+						return nil
+					})
+			})
+		if err != nil {
+			w.Logger.Info(ctx, "Couldn't count restore data size", "location", w.location)
+			continue
+		}
+		w.metrics.SetRemainingBytes(run.ClusterID, location, target.SnapshotTag, target.Keyspace, restorePerLocationDataSize)
```

Comment:
`target.Keyspace` is the glob pattern specified by the `-K` flag, not the list of all restored keyspaces.
Maybe `Units` from `RestoreRun` are more useful in this situation?
But still I don't understand the need to label those metrics by keyspace slice.

Reply:
> target.Keyspace is the glob pattern specified by the -K flag, not the list of all restored keyspaces.

Yes, that's true.
See this comment https://github.com/scylladb/scylla-manager/pull/3331#discussion_r1146788445